modules/account-lib/test/unit/coin/eth/transactionBuilder/walletInitialization.ts

```diff
@@ -71,6 +76,33 @@ describe('Eth Transaction builder wallet initialization', function () {
       txJson.gasPrice!.should.equal('10');
       should.equal(txJson.nonce, 1);
       should.equal(txJson.chainId, 42);
+      should.equal(txJson.v, undefined);
+    });
+
+    it('a wallet initialization transaction, enable v', async () => {
```

Comment:
please add an additional check to make sure this enabling v field check is turned on by default for all eth and then not for rbtc

Reply:
The default value was disabled. I have just changed. But I don't add more test only fix the one exists. You can find test case where no flag is setting and V field is expected.

Reply:
the test cases purpose is to enforce that derived eth coins enforce checking the V value